packages/expo-media-library/src/next/index.ts

```diff
@@ -9,15 +9,57 @@ export * from './MediaLibraryNext.types';
 export class Query extends ExpoMediaLibraryNext.Query {}
 
 export class Asset extends ExpoMediaLibraryNext.Asset {
+  /*
+   * A static function. Creates a new asset from a given file path.
+   * Optionally associates the asset with an album. On Android, if not specified, the asset will be placed in the default "Pictures" directory.
+   *
+   * @param filePath - Local filesystem path (e.g., `file:///...`) of the file to import.
```

Comment:
```suggestion
   * @param filePath - Local filesystem path (for example, `file:///...`) of the file to import.
```

We avoid Latin Abbreviations: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations.